app/controllers/entries_controller.rb

```diff
@@ -4,22 +4,32 @@ class EntriesController < ApplicationController
   DATE_FORMAT = "%d/%m/%Y".freeze
```

Comment:
Maybe we could put this code in two seperate controllers? One for HoursEntries and one for MilesEntries